src/ontology/mondo-edit.obo

```diff
@@ -531789,21 +531769,35 @@ is_a: MONDO:0016120 ! myotonic syndrome
 [Term]
 id: MONDO:0056795
 name: X-linked spermatogenic failure 1
+subset: gard_rare {source="MONDO:GARD"}
 subset: gard_rare {source="GARD:8406", source="MONDO:GARD"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: obsoletion_candidate
+subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "DEL CASTILLO syndrome" EXACT [DOID:0050457]
 synonym: "Del Castillo syndrome" RELATED [OMIM:305700]
-synonym: "Germinal cell aplasia" RELATED [OMIM:305700]
-synonym: "Sertoli cell-only syndrome" RELATED [OMIM:305700]
+synonym: "Germinal cell aplasia" EXACT [DOID:0050457, OMIM:305700]
+synonym: "Sertoli cell-only syndrome" EXACT [MONDO:0010595, OMIM:305700]
 synonym: "spermatogenic failure, X-linked, 1" RELATED [MONDO:Lexical, OMIM:305700]
 synonym: "spermatogenic failure, X-linked, type 1" EXACT [MONDORULE:1, OMIM:305700]
```

Comment:
same here, the same synonym, though spelled differently, is both related and exact. Please update as needed.

Reply:
updated to exact synonyms